<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KBudgetViewDecl</class>
 <widget class="QWidget" name="KBudgetViewDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QGroupBox" name="groupBox1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Your budgets</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="KPushButton" name="m_newButton">
            <property name="text">
             <string comment="New budget">New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="m_renameButton">
            <property name="text">
             <string>Ren</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="m_deleteButton">
            <property name="text">
             <string>Del</string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="m_resetButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Res</string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="m_updateButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Upd</string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="K3ListView" name="m_budgetList">
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <property name="showSortIndicator">
           <bool>true</bool>
          </property>
          <property name="fullWidth">
           <bool>false</bool>
          </property>
          <property name="shadeSortColumn">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string comment="Budget name">Name</string>
           </property>
           <property name="clickable">
            <bool>true</bool>
           </property>
           <property name="resizable">
            <bool>true</bool>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Year</string>
           </property>
           <property name="clickable">
            <bool>true</bool>
           </property>
           <property name="resizable">
            <bool>false</bool>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="KMyMoneyAccountTreeBudget" name="m_accountTree" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>5</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_hideUnusedButton">
         <property name="text">
          <string>Hide unused budget categories</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_assignmentBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Assignments</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="textLabel3">
              <property name="text">
               <string>Account</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KLineEdit" name="m_leAccounts">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1">
              <property name="text">
               <string comment="Budget total">Total</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="kMyMoneyEdit" name="m_accountTotal" native="true">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="calculatorButtonVisibility" stdset="0">
               <bool>false</bool>
              </property>
              <property name="resetButtonVisibility" stdset="0">
               <bool>false</bool>
              </property>
              <property name="readOnly" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="m_cbBudgetSubaccounts">
            <property name="text">
             <string>Include subaccounts</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="KBudgetValues" name="m_budgetValue" native="true"/>
            </item>
            <item>
             <spacer name="spacer2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>K3ListView</class>
   <extends>Q3ListView</extends>
   <header>k3listview.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyAccountTreeBudget</class>
   <extends>QWidget</extends>
   <header>../widgets/kmymoneyaccounttreebudget.h</header>
  </customwidget>
  <customwidget>
   <class>kMyMoneyEdit</class>
   <extends>QWidget</extends>
   <header>kmymoneyedit.h</header>
  </customwidget>
  <customwidget>
   <class>KBudgetValues</class>
   <extends>QWidget</extends>
   <header>kbudgetvalues.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
